Src/GoogleApis.Auth/OAuth2/AuthorizationCodeInstalledApp.cs

```diff
@@ -64,7 +64,7 @@ public ICodeReceiver CodeReceiver
 
             // If the stored token is null or it doesn't have a refresh token and the access token is expired we need 
             // to retrieve a new authorization code.
-            if (token == null || (token.RefreshToken == null && token.IsExpired(flow.Clock)))
+            if (Flow.ShouldForceTokenRetrieval() || token == null || (token.RefreshToken == null && token.IsExpired(flow.Clock)))

```

Comment:
Done. I thought far too long on this but eventually decided to throw it in a separate function to keep it a bit cleaner. I kept and adjusted the in-line comments since I figure they were there for a reason. Part of me wanted to find a way to only check for a token if ```Flow.ShouldForceTokenRetrieval()``` is false anyways, but this should be just fine for now.